masterlist.yaml

```diff
@@ -831,6 +831,24 @@ plugins:
       - 'Order_Of_Sanctity.ESP'
       - 'Walkers of Morrowind.esp'
 
+###### Gameplay - Quests ######
+  ### Rise of House Telvanni ###
+  - name: 'Rise of House Telvanni.esm'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/27545/' ]
+    msg:
+      - <<: *alsoUseX
+        subs: [ '[BOM_ROHT_Patch.ESP](https://www.nexusmods.com/morrowind/mods/49231/)' ]
+        condition: 'file("Rise of House Telvanni.esm") and file("Beautiful cities of Morrowind.ESP")'
```

Comment:
```suggestion
```

You can remove that and replace it with this in the metadata for `Beautiful cities of Morrowind.ESP`

```yaml
  - name: 'Beautiful cities of Morrowind.ESP'
    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
    after:
      - 'Better Landscapes Stonewood Pass (RP Edit).esp'
      - 'Morrowind Anti-Cheese.esp'
    msg:
      - <<: *patchIncluded
        subs: [ 'Rise of House Telvanni' ]
        condition: 'active("Rise of House Telvanni.esm") and not active("BOM_ROHT_Patch.ESP")'
```